### ISSUE TITLE
Manifest Unavailable 

### ISSUE THREAD

**Issue by SpeedrunnerG55** [ValveSoftware/steam-for-linux#9468]
#### Your system information

* Steam client version (build number or date):  Jan 16 2022, at 17:34:49
* Distribution (e.g. Ubuntu): Ubuntu 22.04.2 LTS (64-bit)
* Opted into Steam client beta?: [Yes/No] Yes
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large code pastes as a [Github Gist](https://gist.github.com/)

ever since switching to Proton 8, I have been unable to download any title that requires Proton for me to run. I tried switching back to 7, but I still can't download Proton or the Linux runtime - Sniper.

I get either 
"connection time out"
or 
"manifest unavailable" 
or
"no internet connection"
errors.

some errors I found in the console when trying to download the Linux runtime or Proton

```
Could not connect to X session manager: None of the authentication protocols specified are supported
Could not connect to X session manager: None of the authentication protocols specified are supported
GameAction [AppID 218230, ActionID 3] : LaunchApp changed task to DownloadingDepots with ""
GameAction [AppID 218230, ActionID 3] : LaunchApp changed task to DownloadingDepots with ""
ExecuteSteamURL: "steam://open/downloads"
CAPIJobRequestUserStats - Server response failed 2
CAPIJobRequestUserStats - Server response failed 2
CAPIJobRequestUserStats - Server response failed 2
CAPIJobRequestUserStats - Server response failed 2
```

My internet connection for everything else and even games that don't need Steam play works just fine; I can download any game that runs Nativity just fine. I can not download the Linux runtime or Proton

expected behavior:
the Linux runtime and proton download and the game runs

current behavior:
steam fails to download the Linux runtime or proton and does not get to run the game

#### Steps for reproducing this issue:

1.  switch proton version
2.  open any game that requires Steam Play
3.  steam is unable to download the Linux runtime or proton

**Comment by kisak-valve** [comment]
Hello @SpeedrunnerG55, can you reproduce the issue, then share `~/.steam/root/logs/content_log.txt`?

**Comment by SpeedrunnerG55** [comment]
[content_log.txt](https://github.com/ValveSoftware/steam-for-linux/files/11412655/content_log.txt)

**Comment by rudilee** [comment]
I got this exact same error, got "manifest unavailable" for all proton related downloads, I look at the log, and got the same 404 not found logs.

**Comment by 0xb7767000** [comment]
I'm on Manjaro 22.1.1 and have the same issue. It started on 4th of May, and persists ever since. It also looks like the "manifest unavailable" and "connection time out" errors are pretty much interchangeable.

**Comment by rudilee** [comment]
I got a workaround for this problem by completely reinstalling steam, first I uninstall the steam package, then remove the `~/.steam` and `~/.local/share/Steam`  directories which also remove all the games files too, then reinstall the steam package.

**Comment by SpeedrunnerG55** [comment]
I didn't want to redownload everything. so this is a solution that worked for me, that I got from chat-GPT 😂.
Hope this works for everyone else with this issue

TLDR: I just had to clear the Package directory and delete the appcache directory. and it worked

```
[notes for Ubuntu users I put in brackets] 

If you prefer not to delete everything and want to try deleting specific files to fix the "manifest unavailable" error on Steam, you can try the following steps:

Exit Steam: Make sure Steam is not running in the background.

Delete the "appcache" folder: Go to the Steam installation directory on your computer. By default, it is located in "C:\Program Files (x86)\Steam" on Windows or "~/Library/Application Support/Steam" on macOS. Within the Steam directory, find the "appcache" folder and delete it. This folder contains temporary files that can sometimes cause issues.

[~/.steam/appcache on Ubuntu]

[skip step below]

Delete the "ClientRegistry.blob" file: While still in the Steam directory, look for the "ClientRegistry.blob" file and delete it. This file stores various client-related information and deleting it can resolve certain issues.

[~/.steam/Package on Ubuntu]

Clear the "Package" folder: In the Steam directory, locate and open the "Package" folder. This folder contains a list of numbered files representing different packages. Delete all the files within the "Package" folder.

After deleting these specific files, relaunch Steam and see if the "manifest unavailable" error persists. These steps essentially clear temporary and cached files that may be causing conflicts. However, keep in mind that these files are part of Steam's configuration and deleting them may result in Steam having to update or re-download certain files. If you're unsure, it's always recommended to back up any important files or consult with Steam Support for further assistance.
```
